### PR TITLE
Staging spotify-api and text-wrap

### PR DESCRIPTION
**Pull request by Faro0q**
Added text wrapping and text is cleared in the textbox when user submits the pweet.
image

### CODE REVIEW

File: client/src/components/music/SpotifyLogin.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import NavBar from "../NavBar";
+import { constants, errors, routes } from "../../Common";
+import axios from "axios";
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles({
```
Comment (wcygan) [inline client/src/components/music/SpotifyLogin.js:7]: hmmm why are we doing inline styling here instead of a separate file

Comment (Faro0q) [reply client/src/components/music/SpotifyLogin.js:7]: Will change it once spotify is ready dw

**Comment by Faro0q** [comment]
Landing page: 
image

Login with spotify:
image

Once logged in: 
image

Song search: 
image

Playing a song: 
image

If no lyrics found: 
image

**Comment by wcygan** [comment]
this is pretty sweet -- ship it B)

**Pull request merged**